GRIB2_Template_4_146_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,39 @@
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,23,1,Type of first fixed surface,(see Code table 4.5),,4.5,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,24,1,Scale factor of first fixed surface,,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,25-28,4,Scaled value of first fixed surface,,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,29,1,Type of second fixed surface,(see Code table 4.5),,4.5,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,30,1,Scale factor of second fixed surface,,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,31-32,4,Scaled value of second fixed surface,,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,35-36,2,Verification score,(see Code table 4.120),,4.120,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,37,1,Type of reference dataset for verification,(see Code table 4.121),,4.121,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,38,1,Type of statistical processing over vertical for verification,(see Code table 4.10),,4.10,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,39,1,Type of threshold operator for verification score,(see Code table 4.91),,4.91,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,40,1,Kind of additional arguments for verification score,(see Code table 4.122),,4.122,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,41,1,Number of additional arguments for verification - NA,,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time, , ,The next two entries are repeated NA times na=0: NA - can be 0 times if no parameters are necessary (na=0),,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,42 + (na-1)*5,1,Scale factor of additional argument for verification,,,,,Operational
+Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,43 + (na-1)*5 to 47+(na-1)*5,4,Scaled value of additional argument for verification,,,,,Operational
```

Comment:
```suggestion
Verification scores for analysis or forecast at a horizontal level or in a horizontal layer at a point in time,(43 + (na-1)*5)-(47+(na-1)*5),4,Scaled value of additional argument for verification,,,,,Operational
```

Reply:
another option is to use brackets instead of parens.
[43+(na-1)*5]-[47+(na-1)*5]

Reply:
use brackets